src/isGoodSimFcnName.m

```diff
@@ -0,0 +1,35 @@
+function goodName = isGoodSimFcnName(subsystem, simulinkFcnName)
+% isGoodSimFcnName          Checks to see if a Simulink-fcn name clashes with
+%                           any other Simulink-fcns in scope at the current
+%                           subsystem level
+% Inputs:
+%   subsystem               Path of a subsystem where the fcn will be added
+%   simulinkFcnName         Name of the Simulink-function to be created
+%
+% Outputs:
+%   goodName                Name can be used(1) or not(0)
+%
+% Example:
+%   isGoodSimFcnName('SubSystem_Name','SimFcnName')
+%
+%           ans = 1
+
+    % Get callable Simulink-functions at the current scope
+    [~, prototype] = getCallableFunctions(subsystem);
+    
+    result = zeros(1, length(prototype));
+    
+    splitEquals = split(prototype, '= ');
```

Comment:
Could the `getPrototypeName` function be used instead?

Reply:
Yes, I'll add that in